packages/sonarwhal/src/lib/cli/wizards/new-rule.ts

```diff
@@ -354,10 +358,22 @@ const generateRuleFiles = async (destination: string, data) => {
         const rulePath = path.join(ruleFile.destination, `${rule.normalizedName}.ts`);
         // e.g.: rule-ssllabs/tests/ssllabs.ts
         const testPath = path.join(testFile.destination, `${rule.normalizedName}.ts`);
+        // e.g.: rule-typescript-config/docs/is-valid.ts
+        const docPath = path.join(docFile.destination, `${rule.normalizedName}.md`);
 
         await Promise.all([mkdirpAsync(path.dirname(rulePath)), mkdirpAsync(path.dirname(testPath))]);
 
         await Promise.all([writeFileAsync(rulePath, ruleContent), writeFileAsync(testPath, testContent)]);
+
+        if (data.isMulti) {
+            if (data.isMulti) {
```

Comment:
I think you have something wrong here with this double of checking the same thing.

Reply:
Yes @molant, this is an error, it should be just one `if`